README.md

```diff
@@ -805,10 +805,52 @@ method. It takes 3 arguments: the first is the name of the section as
 referred to on the page (sections that appear on multiple pages can be
 named differently). The second argument is the class of which an
 instance will be created to represent the page section, and the third
-argument is a css selector that identifies the root node of the section
+argument the search argument that identifies the root node of the section
```

Comment:
This doesn't quite read correctly, probably should be reverted

Reply:
Sorry, English is not my native language, so I appreciate any suggestions how to improve wording